Fixed issue with incorrect module name conversion

agorbivskyi opened the pull request:
before: Elasticsearch6 -> elasticsearch6
after:  Elasticsearch6 -> elasticsearch-6

Commit:
Fixed issue with incorrect module name conversion:

before: Elasticsearch6 -> elasticsearch6
after:  Elasticsearch6 -> elasticsearch-6

novykov commented:
This condition breaks current functionality (check with a name like TestModule) and does not fix cases like Tinemce3Banner

Commit:
Fixed issue with incorrect module:

Elasticsearch6 -> elasticsearch-6
Tinemce3Banner -> tinemce-3-banner
CatalogWidget -> catalog-widget
Catalog -> catalog

agorbivskyi commented:
Fixed. Checked with following values:

Elasticsearch6 -> elasticsearch-6
Tinemce3Banner -> tinemce-3-banner
CatalogWidget -> catalog-widget
Catalog -> catalog
